make available for kinetic

`catkin build` throws an error:
```
Errors     << canopen_test_utils:make /u/robot/git/care-o-bot/logs/canopen_test_utils/build.make.000.log          
/u/robot/git/care-o-bot/src/canopen_test_utils/src/elmo_console.cpp:2:35: fatal error: canopen_master/master.h: No such file or directory
```
`master.h` is not available under kinetic anymore

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,14 +2,17 @@
 sudo: required 
 dist: trusty 
 language: generic 
-compiler:
-  - gcc
+
 env:
   matrix:
-    - USE_DEB=true  ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - USE_DEB=true  ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - ROS_DISTRO="indigo"  ROS_REPO=ros
+    - ROS_DISTRO="indigo"  ROS_REPO=ros-shadow-fixed
+    - ROS_DISTRO="kinetic" ROS_REPO=ros
+    - ROS_DISTRO="kinetic" ROS_REPO=ros-shadow-fixed
+
 install:
-  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config  
+  - git clone --depth=1 https://github.com/ros-industrial/industrial_ci.git .industrial_ci
+
 script: 
-  - source .ci_config/travis.sh
+  - .industrial_ci/travis.sh
   

--- a/src/elmo_console.cpp
+++ b/src/elmo_console.cpp
@@ -1,5 +1,4 @@
 #include <canopen_master/canopen.h>
-#include <canopen_master/master.h>
 #include <boost/make_shared.hpp>
 #include <iostream>
 

